composer.json

```diff
@@ -4,24 +4,22 @@
     "require":      {
         "php":                          ">=5.4",
         "bldr-io/composer-plugin":      "~1.0.4",
-        "dflydev/embedded-composer":    "dev-master@dev",
-        "composer/composer":            "dev-master@dev",
-        "symfony/console":              "~2.6",
-        "symfony/config":               "~2.6",
-        "symfony/yaml":                 "~2.6",
-        "symfony/dependency-injection": "~2.6",
-        "symfony/expression-language":  "~2.6",
-        "symfony/event-dispatcher":     "~2.6",
-        "symfony/process":              "~2.6",
+        "dflydev/embedded-composer":    "dev-master",
+        "composer/composer":            "dev-master",
+        "symfony/console":              "~2.3",
+        "symfony/config":               "~2.3",
+        "symfony/yaml":                 "~2.3",
+        "symfony/dependency-injection": "~2.3",
+        "symfony/expression-language":  "~2.3",
+        "symfony/event-dispatcher":     "~2.3",
+        "symfony/process":              "~2.3",
         "zendframework/zend-json":      "~2.3",
         "swiftmailer/swiftmailer":      "~5.3",
-        "bldr-io/frontend-block":       "~2.0.0@dev",
-        "richthegeek/phpsass":          "dev-master",
-        "bldr-io/remote-block":         "~2.0",
```

Comment:
Yeah. I figured they are so small, it didnt really hurt to have the be installed with the core.